### PR TITLE
Update legal page and other minor edits for cleanup

### PR DESCRIPTION
**Pull request by rpellows**

### CODE REVIEW

File: docs/guidelines.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ those requirements before an approval.
 
 Developers will follow the [GitHub
 Flow](https://guides.github.com/introduction/flow/) model for PR submissions.
-Howeber, Ocopea branching is not allowed.  GitHub supports cloning a repository
```
Comment (mrmccarthy) [inline docs/guidelines.md:30]: I kinda want Howeber to be a word...

Comment (rpellows) [reply docs/guidelines.md:30]: Thought I fixed that...

**Review by mrmccarthy — APPROVED** [review]
LGTM

File: docs/guidelines.md (hunk 1 above)
Comment (rpellows) [reply docs/guidelines.md:30]: yes, see a later commit
tried to squash but failed to do so (hence 3 commits)
howeber, does it automatically squash when merging?

**Pull request merged**